platform/viewer/src/connectedComponents/ViewerRetrieveStudyData.js

```diff
@@ -12,7 +13,7 @@ const { studyMetadataManager, updateMetaDataManager } = utils;
 class ViewerRetrieveStudyData extends Component {
   static propTypes = {
     studyInstanceUids: PropTypes.array.isRequired,
-    seriesInstanceUids: PropTypes.array,
+    seriesInstanceUIDs: PropTypes.array,
```

Comment:
It's still confusing/odd to me that we're keeping `studyInstanceUids` but changing the casing of `seriesInstanceUIDs`. Unless we're changing the usage throughout, I'd rather we avoid doing this. At a minimum, we should use the same casing rules within the same functions/scope.

Reply:
changed to seriesInstanceUids